index.js

```diff
@@ -0,0 +1,82 @@
+
+var userScore = 0;
+var computerScore = 0;
+
+//Function for game
+function playGame(score) {
+  do {
+    var user = userChoice();
+    var computer = compChoice();
+    var result = compareChoice(user, computer);
+    updateScores(result, 1);
+  }while(userScore < 3 && computerScore < 3);
+  if(userScore > computerScore) {
+    alert("The user has won 3 out of 5 points. So the user wins!");
+  }
+  else {
+    alert("The computer has won 3 out of 5 points. So the computer wins!");
+  }
+}
+
+//Function for user choice
+function userChoice(){
+  var choice = prompt("(r)rock ,(p)paper or (s)scissors?")
+  switch (choice) {
+    case "r":
+    case "rock":
+      return "rock";
+      break;
+    case "p":
+    case "paper":
+      return "paper";
+      break;
+    case "s":
+    case "scissors":
+      return "scissors";
+      break;
+    default:
+      prompt("Please choose (r)rock ,(p)paper or (s)scissors?");
+      return userChoice()
+  }
+}
+
+//Function for computer choice
+function compChoice() {
+  var comp = ["rock", "paper", "scissors"];
+  var compChoice = comp[Math.floor(Math.random() * Math.floor(3))];
+  return compChoice;
+}
+
+//Function to compare choices
+function compareChoice(choice1, choice2) {
+  alert("user chose: " + choice1 + " and the computer chose: " + choice2 + "!");
+  if(choice1 === choice2) {
+    alert("You and the computer chosen the same thing! Go again, no score added!");
```

Comment:
This should say "have chosen" or "chose".

I would also probably combine this with the above alert. You don't want too many alerts, as they can get quite annoying.